phpcs.xml.dist

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ruleset name="loop">
+  <description>PHP CodeSniffer configuration for ereolen base</description>
+
+  <file>sites/all/modules</file>
+  <file>sites/all/themes</file>
+
+  <exclude-pattern>sites/all/default</exclude-pattern>
+  <exclude-pattern>includes/</exclude-pattern>
+  <exclude-pattern>misc/</exclude-pattern>
+  <exclude-pattern>modules/</exclude-pattern>
+  <exclude-pattern>profiles/</exclude-pattern>
+  <exclude-pattern>scripts/</exclude-pattern>
+  <exclude-pattern>themes</exclude-pattern>
+  <exclude-pattern>vendor/</exclude-pattern>
```

Comment:
You're only processing files in `sites/all/{modules,themes}` so it shouldn't be necessary to exclude these folders. Only `sites/all/modules/contrib/` should be excluded.